server.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const { getCatgories } = require('./controllers/getCatagories.controller');
+const app = express();
+
+app.use(express.json());
```

Comment:
Remove app.use(express.json()). Don't need it yet!

Reply:
Ok thanks will do!